blockchain/tests/inherents.rs

```diff
@@ -84,12 +84,14 @@ fn it_can_create_batch_finalization_inherents() {
 
     let (validator_address, _) = active_validators.iter().next().unwrap();
 
+    const EXPECTED_REWARD: u64 = 166_810_895;
```

Comment:
This high number is expected, since in the unit test chain, we have an initial supply of 1.01e12 but a total supply of 2100e12. The remaining supply is 2098.99e+12, which is 500_438_213 shifted 22 to the right. Since the block time is 20 seconds, so we expect the reward to be roughly(!) one third of it, so 166_812_737. Hence, the value seems to be correct.